Userland/Libraries/LibGL/SoftwareGLContext.cpp

```diff
@@ -712,9 +710,8 @@ void SoftwareGLContext::gl_tex_image_2d(GLenum target, GLint level, GLint intern
     RETURN_WITH_ERROR_IF((width & 2) != 0 || (height & 2) != 0, GL_INVALID_VALUE);
     RETURN_WITH_ERROR_IF(border < 0 || border > 1, GL_INVALID_VALUE);
 
-    // TODO: Load texture from the currently active texture unit
-    // This is to test the functionality of texture data upload
-    static_ptr_cast<Texture2D>(m_allocated_textures.find(1)->value)->upload_texture_data(target, level, internal_format, width, height, border, format, type, data);
+    // We gate the texture unit on line 705, so we know that the active Texture Unit is bound to GL_TEXTURE_2D
```

Comment:
Let's not reference line numbers in comments, that's bound to get outdated by next week :)

(also seems wrong already, isn't it 703?)

Reply:
Done :D